src/components/accordion/accordion.tsx

```diff
@@ -1,57 +1,18 @@
-import { Dispatch, FC, Fragment, HTMLAttributes, ReactNode, SetStateAction, useState } from 'react'
+import { Dispatch, FC, Fragment, SetStateAction, useState } from 'react'
 import { cx } from '@linaria/core'
+import { elIsActive } from '../../styles/states'
+import { Icon } from '../icon'
+import { handleKeyboardEvent } from '../../storybook/handle-keyboard-event'
+import { useId } from '../../storybook/random-id'
+import { AccordionProps } from './types'
 import {
   ElAccordionContainer,
   ElAccordionContent,
   ElAccordionItem,
   ElAccordionTitle,
   ElAccordionTitleContent,
   ElAccordionTitleContentWrapper,
-} from './__styles__'
-import { elIsActive } from '../../styles/states'
-import { Icon } from '../icon'
-import { handleKeyboardEvent } from '../../storybook/handle-keyboard-event'
-import { useId } from '../../storybook/random-id'
-
-export interface AccordionBaseProps extends HTMLAttributes<HTMLDivElement> {}
-export interface AccordionBaseItemProps extends HTMLAttributes<HTMLAnchorElement> {}
-
-export interface AccordionTitleItemProps {}
-
-export interface AccordionItemProps {
-  title: string
-  titleItems?: ReactNode[]
-  content: ReactNode
-  onClick?: () => void
-}
-
-export interface AccordionProps extends AccordionBaseProps {
-  items: AccordionItemProps[]
-}
-
-export const AccordionContainer: FC<AccordionBaseProps> = ({ children, ...rest }) => {
-  return <ElAccordionContainer {...rest}>{children}</ElAccordionContainer>
-}
-
-export const AccordionItem: FC<AccordionBaseItemProps> = ({ children, ...rest }) => {
-  return <ElAccordionItem {...rest}>{children}</ElAccordionItem>
-}
-
-export const AccordionTitle: FC<HTMLAttributes<HTMLDivElement>> = ({ children, ...rest }) => {
-  return <ElAccordionTitle {...rest}>{children}</ElAccordionTitle>
-}
-
-export const AccordionTitleContentWrapper: FC<AccordionBaseProps> = ({ children, ...rest }) => {
-  return <ElAccordionTitleContentWrapper {...rest}>{children}</ElAccordionTitleContentWrapper>
-}
-
-export const AccordionTitleContent: FC<AccordionBaseProps> = ({ children, ...rest }) => {
-  return <ElAccordionTitleContent {...rest}>{children}</ElAccordionTitleContent>
-}
-
-export const AccordionContent: FC<AccordionBaseProps> = ({ children, ...rest }) => {
-  return <ElAccordionContent {...rest}>{children}</ElAccordionContent>
-}
+} from './styles'
```

Comment:
**question:** @willmcvay what was your thinking behind leaving these styled components in use instead of the new "atoms" from `accordion.atoms.tsx`? Were you just trying to minimise the amount of changes in the PR?